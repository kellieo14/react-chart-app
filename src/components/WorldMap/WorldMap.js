import React from 'react';
import worldMapImg from '../../images/worldMap.png';
import Col from 'react-bootstrap/Col';
import Container from 'react-bootstrap/Container';
import Row from 'react-bootstrap/Row';
import './WorldMap.css';

function WorldMap() {
  return (
    <Container>
      <Row>
        <Col sm={12}>
          <img alt='world map' className='map' src={worldMapImg}></img>
        </Col>
      </Row>
    </Container>
  );
}

export default WorldMap;