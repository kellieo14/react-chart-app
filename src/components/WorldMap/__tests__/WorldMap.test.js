import React from 'react';
import TestRenderer from 'react-test-renderer';
import WorldMap from '../WorldMap';


describe('WorldMap', () => {
    it('should match snapshot', () => {
        const testRenderer = TestRenderer.create(<WorldMap />);
        expect(testRenderer.toJSON()).toMatchSnapshot();
    });
});