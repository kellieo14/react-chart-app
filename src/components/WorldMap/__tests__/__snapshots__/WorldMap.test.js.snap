// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WorldMap should match snapshot 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-sm-12"
    >
      <img
        alt="world map"
        className="map"
        src="worldMap.png"
      />
    </div>
  </div>
</div>
`;
